components/org.wso2.carbon.identity.scim2.provider/src/main/java/org/wso2/carbon/identity/scim2/provider/resources/GroupResource.java

```diff
@@ -394,6 +395,9 @@ public Response patchGroup(@PathParam(SCIMConstants.CommonSchemaConstants.ID) St
         requestAttributes.put(SCIMProviderConstants.ATTRIBUTES, attribute);
         requestAttributes.put(SCIMProviderConstants.EXCLUDE_ATTRIBUTES, excludedAttributes);
         requestAttributes.put(SCIMProviderConstants.SEARCH, "0");
+
+        SupportUtils.updateIdentityContextFlow(Flow.Name.GROUP_UPDATE);
```

Comment:
Shall we move this to top to be consistent with the updateUser() in UserResource and MeResource

Reply:
Fixed 2f61995f52ef116225b64e484b280d8914d3ced7